.github/workflows/validate-pull-request.yaml

```diff
@@ -0,0 +1,15 @@
+name: PR Validation
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  pr-validation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
this might be a stupid question but: "where" are these actions that these steps are referring to?

Reply:
https://github.com/actions/